Pango-1.0.gir

```diff
@@ -6171,15 +6171,15 @@ the shaping and rendering phases.
 
 Currently, it contains cluster start and color information.
 More attributes may be added in the future.</doc>
-      <field name="is_cluster_start" writable="1" bits="1">
+      <field name="is_cluster_start" writable="1">
         <doc xml:space="preserve">set for the first logical glyph in each cluster.
   (Clusters are stored in visual order, within the cluster, glyphs
   are always ordered in logical order, since visual order is meaningless;
   that is, in Arabic text, accent glyphs follow the glyphs for the
   base character.)</doc>
         <type name="guint" c:type="guint"/>
       </field>
-      <field name="is_color" writable="1" bits="1">
+      <field name="is_color" writable="1">
```

Comment:
One of the two fields has to go away, otherwise the struct has twice the size it should

Reply:
I'm not sure whether it matters at the end (as nothing uses it) other than maybe -sys tests. But should be fixed now